manifests/base/rstudio-buildconfig.yaml

```diff
@@ -0,0 +1,73 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+    component: rstudio-rhel9-image
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/rstudio"
+    opendatahub.io/notebook-image-name: "R Studio"
+    opendatahub.io/notebook-image-desc: "R Studio workbench image, allows to integrated development environment for R, a programming language for statistical computing and graphics."
+  name: rstudio-rhel9
+spec:
+  name: latest
+  lookupPolicy:
+    local: true
```

Comment:
```suggestion
kind: ImageStream
apiVersion: image.openshift.io/v1
metadata:
  annotations:
    opendatahub.io/notebook-image-creator: RHOAI
    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/rstudio"
    opendatahub.io/notebook-image-name: "R Studio"
    opendatahub.io/notebook-image-desc: "R Studio workbench image, allows to integrated development environment for R, a programming language for statistical computing and graphics."
  name: rstudio-rhel9
  labels:
    opendatahub.io/dashboard: 'true'
    opendatahub.io/notebook-image: 'true'
spec:
  lookupPolicy:
    local: true
  tags:
    - name: latest
      annotations:
        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
        opendatahub.io/notebook-python-dependencies: '[{"name":"r-studio","version":"4.3"}]'
      referencePolicy:
        type: Source
```

Reply:
Great, thank you so much for the suggestion! I added it and tested, works as expected now.
![image](https://github.com/red-hat-data-services/notebooks/assets/42587738/8000162b-ab16-4ccf-aa32-6d886c085e7f)